tests/dummy/app/templates/snippets/buttons-dropdown.hbs

```diff
@@ -0,0 +1,9 @@
+{{md-btn-dropdown text='Button' activates="dropdown-target"}}
```

Comment:
Ok I'll try that



> Op 4 jul. 2015 om 00:26 heeft Mike North <notifications@github.com> het volgende geschreven:
> 
> In tests/dummy/app/templates/snippets/buttons-dropdown.hbs:
> 
> > @@ -0,0 +1,9 @@
> > +{{md-btn-dropdown text='Button' activates="dropdown-target"}}
> Until ember v2.1 or v2.2, we will continue to have only one "rich" content projection tool, and that's {{yield}}. In the case of a simple button, it makes sense to use it for the content of the button (i.e., the label) since that's really all we care about.
> 
> In this case, I think that what will most often be customized will be the contents of the dropdown, and thus it makes the most sense to use the yield for it.
> 
> —
> Reply to this email directly or view it on GitHub.
>